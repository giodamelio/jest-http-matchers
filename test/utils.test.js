const { createMessage } = require('../src/utils');

it('create message', () => {
  expect(
    createMessage(
      'Expected test to do a thingy (totally, I promise)',
      'the.name.of.the.matcher',
      'expected thingy',
      'received thingy'
    )
  ).toMatchSnapshot();
});
