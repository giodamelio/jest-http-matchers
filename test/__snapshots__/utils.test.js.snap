// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create message 1`] = `
"[2mexpect([22m[31mreceived[39m[2m)the.name.of.the.matcher([22m[32mexpected[39m[2m)[22m

Expected test to do a thingy (totally, I promise)
  [32m\\"expected thingy\\"[39m
Received:
  [31m\\"received thingy\\"[39m"
`;
